javaee/tests/src/test/java/org/jboss/forge/addon/javaee/security/ui/AbstractSecurityCommandTest.java

```diff
@@ -0,0 +1,63 @@
+package org.jboss.forge.addon.javaee.security.ui;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.forge.addon.javaee.ProjectHelper;
+import org.jboss.forge.addon.javaee.security.AuthenticationMethod;
+import org.jboss.forge.addon.javaee.servlet.ServletFacet;
+import org.jboss.forge.addon.javaee.servlet.ServletFacet_2_5;
+import org.jboss.forge.addon.javaee.servlet.ServletFacet_3_0;
+import org.jboss.forge.addon.javaee.servlet.ServletFacet_3_1;
+import org.jboss.forge.addon.parser.java.projects.JavaWebProjectType;
+import org.jboss.forge.addon.projects.Project;
+import org.jboss.forge.addon.ui.controller.CommandController;
+import org.jboss.forge.addon.ui.test.UITestHarness;
+import org.jboss.forge.arquillian.AddonDependencies;
+import org.jboss.forge.arquillian.archive.AddonArchive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.junit.Before;
+
+import javax.inject.Inject;
+
+public abstract class AbstractSecurityCommandTest
+{
+   @Inject
+   protected ProjectHelper projectHelper;
```

Comment:
I need the projectHelper in this class in order to create a temporary project in the setup method. Then it is a matter of preference whether I use that same object to install the facet or inject another dependency (the facet factory). I chose the first option to keep the code shorter